renderer/components/Channels/ChannelCreateForm.js

```diff
@@ -33,9 +33,13 @@ import {
  * Animation to handle showing/hiding the amount fields.
  */
 const ShowHide = Keyframes.Spring({
-  show: async next => {
+  show: async (next, cancel, ownProps) => {
     await next({ pointerEvents: 'auto' })
     await next({ opacity: 1 })
+    // Focus the amount input if the current step is the form step.
+    if (ownProps.context.state.step === 'form') {
+      ownProps.context.focusAmountInput()
```

Comment:
Do you think `Spring keyframes` is an appropriate place for this? Maybe something like `cdu` would be better ?

Reply:
The issue is that we can only focus it once it has been rendered, but it only gets rendered after the show animation has completed. This is the only place that we get notified of the animation having completed. ComponentDidUpdate does not get called.